app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, source_file, target_file = command.split()
+    if cmd != "cp" or source_file == target_file:
+        return
+    try:
+        with (open(source_file, "r") as source,
```

Comment:
The `open` function could raise an `IOError` if the file cannot be opened for some reason (e.g., insufficient permissions, disk space, etc.). Consider handling this exception as well.